src/language/ast.js

```diff
@@ -448,6 +448,7 @@ export type ScalarTypeDefinitionNode = {
   +loc?: Location,
   +description?: StringValueNode,
   +name: NameNode,
+  +type?: NamedTypeNode,
```

Comment:
I think this key should be more descriptive. A type having a field called "type" is confusing

Reply:
Fully agree. How about `serializableAs`?

Reply:
@leebyron Given current syntax `scalar Url as String`.
How about `asType`?